Add coopdevs call notes (#40)
hyphacoop/organizing: 2019-04-25-coopdevs-chat.md

---
source_url: https://hackmd.io/V9joJrlrSvWed0-ez2Phag
---
# 2019-04-25 Toronto Workers Co-op: CoopDevs Chat

<sup>[from template][template]</sup>

[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]

Time: Thu, Apr 25 @ 10-11am ET  
Location: https://meet.jit.si/offline  
Attending: patcon, dcwalk, ben&ferran, pau

[TOC]

## Agenda

- Intros

>     - Hypha Workers Co-op

Reviewer: ```suggestion
    - Hypha Worker Co-op
```

Author: gah... i've definitely been one of the ppl sowing confusion with using the wrong word (had a misunderstanding earlier that's hard to break!)

    - totally separate legal entities. informal agreement between assoc members and co-op. but same project and single brand.
        - one logo → http://coopdevs.org/
        - same communication channels
    - revenue
        - assoc shouldn't need much money to run. to be associate it's just 10€. But coop feeds or helps it. stickers, events, etc.
        - coop has customers and services and some grants in order to pay salaries (non-profit).
- Most people are full-time or near
- 60% billable work, 40% internal/donated/volunteer work
    - "like a donation of cooperative to project"
    - non-paid work, non-client work: work on community, communication "re-productive" :)
        - "trying to get rid of this work"? Couldn't hear well. [name=patcon]
    - timeoverflow example: 20h a month as a donation. One city council is paying for some feature but we out-sourced it from the coop.
- Grant as a means to start and consolidate projects.
    - ERP (resource planning) example: goal is that next year should be in market and switch to customer revenue v. grant
    - lucky: city gov and catalan govt eager to fund coop and solidarity econ

>         - [research group at catalon university](https://www.uoc.edu/portal/en/in3/recerca/grups/digital_commons) are big fans of platform coop and them (pilot proj & EU funding)

Reviewer: ```suggestion
        - [research group at Catalunya university](https://www.uoc.edu/portal/en/in3/recerca/grups/digital_commons) are big fans of platform coop and them (pilot proj & EU funding)
```

        - for Katuma digital platform coop
        - coopdevs initated it as a kind of little incubator, each member has many projects

> - org has accumulated many projects

Reviewer: Got permission to share this :)

```suggestion

![coopdevs projects](https://i.imgur.com/dozQl5n.jpg)

- org has accumulated many projects (above diagram shows members responsible for them)
```

Author: Niiiiice. Thanks Ben! Curious: know what green dots and exclamation marks mean?

Author: Also, feels maybe worth pointing back to the fact that ferran+antonio (who have much less lines of responsibility) are non-member workers, rather than member-owners that others are.

    - examples: internal tech, communication (social nw), katuma, timeoverflow...
    - coop telecom = 30% income
    - but most weight in income: SomConnexió (telecom), OpenFoodNetwork
- tooling
    - gitlab.com + their ci (slow transition from github)
    - appear in → jitsi
    - bitwarden for passwd sharing
    - chat in telegram (plan to transition) riot/matrix.org for "pam a pam" project
    - difficulty to find the time to know the tools and more to switch.
    - discourse forum for the coopdevs community
    - time tracking: trello or [Toggl](https://www.toggl.com/trello-time-tracking/) are we winning/losing money? How much productive/reproductive work are we doing? Which projects are we investing more. We are using closed source toggl.com . "our" tool [Odoo](https://www.odoo.com/) has some feature for time tracking but it's a little clumsy.
    - Odoo is an enterprise solution. Focused on CRM, invoicing, accounting, ...  and has miriads of modules. Companies try to fit everything inside it.
- making decisions

>     - initial difference on contracts of wokers with owner-workers in the legal and responsability plane, but info and decision making is shared. 

Reviewer: ```suggestion
    - initial difference on contracts of workers with owner-workers in the legal and responsibility plane, but info and decision making is shared
```

        - coop has customers and services and some grants in order to pay salaries (non-profit).
- Most people are full-time or near
- 60% billable work, 40% internal/donated/volunteer work
    - "like a donation of cooperative to project"
    - non-paid work, non-client work: work on community, communication "re-productive" :)
        - "trying to get rid of this work"? Couldn't hear well. [name=patcon]
    - timeoverflow example: 20h a month as a donation. One city council is paying for some feature but we out-sourced it from the coop.
- Grant as a means to start and consolidate projects.
    - ERP (resource planning) example: goal is that next year should be in market and switch to customer revenue v. grant
    - lucky: city gov and catalan govt eager to fund coop and solidarity econ
        - [research group at Catalunya university](https://www.uoc.edu/portal/en/in3/recerca/grups/digital_commons) are big fans of platform coop and them (pilot proj & EU funding)
        - for Katuma digital platform coop
        - coopdevs initated it as a kind of little incubator, each member has many projects

![coopdevs projects](https://i.imgur.com/dozQl5n.jpg)

> - org has accumulated many projects (above diagram shows members responsible for them)

Author: ```suggestion
- org has accumulated many projects
  - above diagram shows members responsible for them
  - note: ferran+antonio (less responsibilities) are non-member staff; rest are member-owners
```

Author: Good to merge after this
